prepare-wp-tests.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash -e
+
+# Convenient functions for printing colored text
+function green {
+  # green text to stdout
+	echo "$@" | sed $'s,.*,\e[32m&\e[m,' | xargs -0 printf
+}
+
+function yellow {
+	# yellow text to stderr
+	echo "$@" | sed $'s,.*,\e[33m&\e[m,' | >&2 xargs -0 printf
+}
+
+function red {
+	# red text to stderr
+	echo "$@" | sed $'s,.*,\e[31m&\e[m,' | >&2 xargs -0 printf
+}
+
+# Check Bash version
+if [[ "${BASH_VERSINFO[0]}" -lt 4 ]]; then
+    red "Error: Bash version 4 or later is required for this script."
+    exit 1
+fi
+
+# Define defaults
+declare -A defaults
+defaults=(
+    ["WP_VERSION"]="latest"
+    ["WP_CORE_DIR"]="/tmp/wordpress"
+    ["TMPDIR"]="/tmp"
+    ["WP_TESTS_DIR"]="/tmp/wordpress-tests-lib"
+    ["WP_MULTISITE"]="0"
+    ["DB_NAME"]="wordpress_unit_tests"
+    ["DB_USER"]="root"
+    ["DB_PASS"]=""
+    ["DB_HOST"]="localhost"
+    ["SKIP_DB_CREATE"]="true"
+    ["CLONE_VIP_MU_PLUGINS"]="true"
+    ["CLONE_VIP_MU_PLUGINS"]="true"
+)
+
+check_env_vars() {
+    for var in "${!defaults[@]}"; do
+        if [ -z "${!var}" ]; then
+            yellow "Warning: environment variable $var is unset. Falling back to default: ${defaults[$var]}"
+        fi
+        export "${var}"="${!var:-${defaults[$var]}}"
+    done
+}
+
+# Check if environment variables are set
+check_env_vars
+
+# Install WordPress to WP_CORE_DIR
+# We might be able to use mantle's version if they implement a way to install_test_suite
```

Comment:
@srtfisher , does this sound correct?

Reply:
I have a PR going that might work well to combine/condense some of this. Stay tuned...